packages/lib-utils/src/app/AppInitSDK.tsx

```diff
@@ -27,7 +30,7 @@ type AppInitSDKProps = {
  * ```tsx
  * return (
  *  <Provider store={store}>
- *   <AppInitSDK configurations={{ appFetch, apiDiscovery }}>
+ *   <AppInitSDK configurations={{ appFetch, pluginStore, wsAppSettings }}>
```

Comment:
```suggestion
 *   <AppInitSDK configurations={{ appFetch, pluginStore, wsAppSettings, apiDiscovery }}>
```

Reply:
It's optional, I figured since this is going to grow any number of ways, required props can be the JS Doc... an example does not replace everything.

Reply:
> It's optional

Right, I missed that, sorry.

> required props can be the JS Doc

:+1: